components/tyk-gateway/values.yaml

```diff
@@ -196,6 +197,8 @@ gateway:
   # - name: TYK_GW_HTTPSERVEROPTIONS_SSLINSECURESKIPVERIFY
   #   value: "true"
   extraEnvs: []
+  extraEnvsFromSecret: []
+  extraEnvsFromConfigMap: []
```

Comment:
why do we need them?

Reply:
Fixed

Reply:
can you please also delete them from `values.yaml` file?

Reply:
Done